.github/workflows/REVIEWING.md

```diff
@@ -0,0 +1,13 @@
+# Reviewing pull requests
+
+Thanks to GitHub Actions, an artifact (=zip file) of the rendered protocol is now automatically created on each pull request.
+
+Reviewers can proceed like that:
+
+1) On the PR page, you can find a "details" link under "checks - On PR, build the site and ...". Go there, click on the top link in the left sidebar, and download the generated artifact at the bottom of the page.
+2) Decompress it and make sure the target directory is called 'protocols' (you may need to rename it)
+3) From the parent directory (just above tutorials), run `python -m http.server 8887`, _or_ launch the Google Chrome [Web Server app](https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb) and point it at the parent directory.
```

Comment:
This seems complicated.  Why not use the standard GitHub tools?

Reply:
I need to simplify this. This is copy pasted from the tutorials repo (hugo website) and can be much simplified.

Reply:
see 
458044b